examples/js/api-lifecycle.test.js

```diff
@@ -0,0 +1,173 @@
+'use strict'
+
+const shared = require('./shared')
+const request = require('supertest')(shared.apiUrl)
+
+const axios = require('axios')
+const { describe, expect } = require('@jest/globals')
```

Comment:
Should also add the `it` dependency, to keep some linters from complaining about its absence.